tests/unit/test_modifications.py

```diff
@@ -0,0 +1,257 @@
+import numpy as np
+import pytest
+
+from tests.conftest import RINGTEST_DIR
+
+from neurodamus.core.configuration import ConfigurationError, SimConfig
+from neurodamus.modification_manager import ModificationManager
+from neurodamus.node import Neurodamus, Node
+
+SIMULATION_CONFIG_FILE = RINGTEST_DIR / "simulation_config.json"
+
+
+def test_applyTTX():
+    """
+    A test of enabling TTX with a short simulation.
+    As Ringtest cells don't contain mechanisms that use the TTX concentration
+    to enable/disable sodium channels, no spike change is expected.
+    Instead, we check that all sections contain TTXDynamicsSwitch after modification
+    """
+
+    # NeuronWrapper needs to be imported at function level
+    from neurodamus.core import NeuronWrapper as Nd
+
+    n = Node(str(SIMULATION_CONFIG_FILE))
```

Comment:
why do you use Node instead of Neurodamus?

Reply:
Because I want to check the cell objects before and after `enable_modifications` . `Neurodamus` class will directly execute `enable_modifications`.